src/python/pants/backend/experimental/makeself/BUILD

```diff
@@ -0,0 +1,11 @@
+# Copyright 2023 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+# NOTE: Sources restricted from the default for python_sources due to conflict with
```

Comment:
Hm, other `BUILD` files don't seem to need to do this; why is this one special? (Example I checked: https://github.com/pantsbuild/pants/blob/db14a1095d6b0ed4a36b0bf84ee2cbd07085da07/src/python/pants/backend/experimental/adhoc/BUILD)